README.md

```diff
@@ -319,22 +319,20 @@ does not build `extensions` and `distribution` modules.
 
 ### Testing
 
-Take into account that the default build executes thousands of tests which may
-take a considerable amount of time. Hazelcast has 3 testing profiles:
+Take into account that the default build executes thousands of tests which may take a considerable amount of time.
+Hazelcast has 3 testing profiles:
 
-* Default: Type `mvn test` to run quick/integration tests (those can be run
-  in parallel without using network).
-* Slow Tests: Type `mvn test -P slow-test` to run tests that are either slow
-  or cannot be run in parallel.
-* All Tests: Type `mvn test -P all-tests` to run all tests serially using
-  network.
+* Default: Type `mvn test` to run quick/integration tests (those can be run in parallel without using network).
+* Slow Tests: Type `mvn test -P slow-test` to run tests that are either slow or cannot be run in parallel.
+* All Tests: Type `mvn test -P all-tests` to run all tests serially using network.
+
+Some tests require Docker to run. Set `-Ddisable.docker.tests` system property to ignore them
 
 ## Trigger Phrases in the Pull Request Conversation
 
-When you create a pull request (PR), it must pass a build-and-test
-procedure. Maintainers will be notified about your PR, and they can
-trigger the build using special comments. These are the phrases you may
-see used in the comments on your PR:
+When you create a pull request (PR), it must pass a build-and-test procedure. Maintainers will be notified about your
+PR, and they can trigger the build using special comments. These are the phrases you may see used in the comments on
+your PR:
```

Comment:
I think this was formatted to 80 chars, let's keep that.

Reply:
IntelliJ formatting has kicked in :/